BuildLoader.py

```diff
@@ -44,12 +44,12 @@ def rebuild_basetools ():
 			ret = subprocess.call(['make', '-C', 'BaseTools'])
 
 	elif os.name == 'nt':
+		if 'PYTHON_HOME' not in os.environ:
+			os.environ['PYTHON_HOME'] = 'C:\\Python27'
```

Comment:
Assumes this directory exists. I think this is a sane default however.

Reply:
The problem is PYTHON_HOME might not have been set by default.  It was my case on my platform.